contracts/managers/price-triggers/MovingAverageToAssetPriceCrossoverTrigger.sol

```diff
@@ -0,0 +1,90 @@
+/*
+    Copyright 2019 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+pragma experimental "ABIEncoderV2";
+
+import { IPriceTrigger } from "./IPriceTrigger.sol";
+import { IOracle } from "../../meta-oracles/interfaces/IOracle.sol";
+import { IMetaOracleV2 } from "../../meta-oracles/interfaces/IMetaOracleV2.sol";
+
+
+/**
+ * @title MovingAverageToAssetPriceCrossoverTrigger
+ * @author Set Protocol
+ *
+ * Implementing the IPriceTrigger interface, this contract is queried by a
+ * RebalancingSetToken Manager to determine the amount of base asset to be
+ * allocated to by checking if the the trading pair price is above or below
+ * a simple or exponential moving average.
+ *
+ * Below the moving average means the RebalancingSetToken should be in the
+ * quote asset, above the moving average the RebalancingSetToken should be
+ * in the base asset.
+ */
+contract MovingAverageToAssetPriceCrossoverTrigger is
```

Comment:
To generalize the naming a bit, perhaps the contract name can be cleaned up to `MetaOracleCrossoverTrigger`. 

Don't need to act on the below, but wanted to note:
Even better if we can somehow generalize it to just `CrossoverTrigger`, where the interfaces of `IMetaOracleV2` and `IOracle` ends up being the same.

Reply:
Might be a bit early, but I wonder when it makes sense to split up this so the code is:

`Trigger` -> `Allocator`